docs/getting_started/install_ez_platform.md

```diff
@@ -1,12 +1,217 @@
+
+From the folder into which you downloaded eZ Platform, run:
+
+``` bash
+composer install
+```
+
+### B. Create project with Composer
+
+You can also use Composer to instantly create a project in the current folder with all the dependencies:
+
+``` bash
+composer create-project --no-dev --keep-vcs ezsystems/ezplatform .
+```
+
+!!! tip
+
+    You can also select a specific version to install, for example:
+
+    ``` bash
+    composer create-project --no-dev --keep-vcs ezsystems/ezplatform . v2.2.2
+    ```
+
+## Provide installation parameters
+
+After a moment the installer will ask you to provide a few parameters:
+
+1. Choose a [secret](http://symfony.com/doc/current/reference/configuration/framework.html#secret); it should be a random string, made up of up to 32 characters, numbers, and symbols. This is used by Symfony when generating [CSRF tokens](http://symfony.com/doc/current/book/forms.html#forms-csrf), [encrypting cookies](http://symfony.com/doc/current/cookbook/security/remember_me.html), and for creating signed URIs when using [ESI (Edge Side Includes)](http://symfony.com/doc/current/book/http_cache.html#edge-side-includes).
+1. You can accept the default options for `database_driver`, `database_host` and `database_port`.
+1. Select a `database_name` or accept the default one.
+1. Provide your `database_user` and `database_password`.
+
+!!! tip
+
+    If you want to change any of these parameters later, you can do it in `app/config/parameters.yml`.
+
+## Create database
+
+!!! tip
+
+    You can omit this step. If you do not create a database now, it will be created automatically in the next step.
+
+Create a database. Run the following command inside MySQL Shell:
+
+``` bash
+CREATE DATABASE ezplatform CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_520_ci;
+```
+
+## Install eZ Platform
+
+Install eZ Platform with:
+
+``` bash
+composer ezplatform-install
+```
+
+This command will also create a database, if you had not created it earlier.
+Before executing it make sure that the database user has sufficient permissions.
+
+If Composer asks for your token, you must log in to your GitHub account and generate a new token
+(edit your profile, go to Developer settings > Personal access tokens and Generate new token with default settings).
+This operation is performed only once when you install eZ Platform Enterprise Edition for the first time.
+
+## Use PHP's built-in server
+
+At this point you can use PHP's built-in server: `php bin/console server:start`.
+
+If you want to use an Apache web server, you need to [set up directory permissions](#set-up-permissions) and [prepare a virtual host](#set-up-virtual-host).
+
+## Prepare installation for production
+
+To use eZ Platform with an HTTP server, you need to [set up directory permissions](#set-up-permissions) and [prepare a virtual host](#set-up-virtual-host).
+
+### Set up permissions
+
+The web user must be the owner of all your files (for example with the `www-data` web user):
```

Comment:
For dev usage yes, for prod this is a security issue. So we'll need to rephrase here to stay clear of that.